tests/test_lightning.py

```diff
@@ -109,3 +109,18 @@ def test_lightning_integration(tmp_dir):
     assert os.path.join(scalars, "train", "epoch", "loss.tsv") in logs
     assert os.path.join(scalars, "train", "step", "loss.tsv") in logs
     assert os.path.join(scalars, "epoch.tsv") in logs
+
+
+def test_lightning_default_dir(tmp_dir):
+    model = LitXOR()
+    # If `dir` is not provided handle it properly, use default value
+    dvclive_logger = DVCLiveLogger("test_run")
+    trainer = Trainer(
+        logger=dvclive_logger,
+        max_epochs=2,
+        enable_checkpointing=False,
+        log_every_n_steps=1,
+    )
+    trainer.fit(model)
+
+    assert os.path.exists("dvclive")
```

Comment:
```suggestion
    assert (tmp_dir /"dvclive").is_dir()
```

Reply:
is it actually needed? some other tests should be fixed as well .. @skshetry please feel free to commit this

Reply:
Let's leave it out then. We should use `pathlib.Path` when we can, as it comes from fixture itself.